lib/heartcheck.rb

```diff
@@ -38,7 +41,8 @@ def logger
     #   end
     #
     # @return [void]
-    def setup
+    def setup(options = {})
+      @hash_formatter = options.fetch(:hash_formatter, false)
```

Comment:
I think we should make setup feel like using the threaded executor, a method that you can call inside your setup block to set this variable, making things more standard